src/cargo/ops/cargo_install.rs

```diff
@@ -27,23 +28,12 @@ struct CrateListingV1 {
     v1: BTreeMap<PackageId, BTreeSet<String>>,
 }
 
-struct Transaction {
```

Comment:
I think we may still want this transaction type in some form. Before we write out the metadata of *new* binaries if an error happens we need to remove them. If we overwrite something an then an error happens, it's fine to throw our hands up in the air